src/components/HowItWorksCard/HowItWorksCard.tsx

```diff
@@ -8,23 +8,23 @@ const HowItWorksCard = () => (
   <Card>
     <CardBody className="how-it-works">
       <TextContent>
-        <Text component={TextVariants.h1}>How it works</Text>
-        <Text>
+        <h1>How it works</h1>
```

Comment:
Any reason why are switching from PF components to the default elements?

Reply:
Updated this, along with other instances, to make use of PF `Text`.